Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headsets.yml

```
@@ -42,6 +42,43 @@
     sprite: DeltaV/Clothing/Ears/Headsets/syndicate_listening.rsi
   - type: Clothing
     sprite: DeltaV/Clothing/Ears/Headsets/syndicate_listening.rsi
+
+- type: entity
+  parent: ClothingHeadset
+  id: ClothingHeadsetJustice
+  name: justice headset
+  description: This is used by the justice department.
+  components:
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyJustice
+      - EncryptionKeySecurity
+      - EncryptionKeyCommon
+  - type: Sprite
+    sprite: DeltaV/Clothing/Ears/Headsets/legal.rsi
+    state: icon
+  - type: Clothing
+    sprite: DeltaV/Clothing/Ears/Headsets/legal.rsi
+
+- type: entity
+  parent: ClothingHeadset
+  id: ClothingHeadsetCJ
```

If this is an over-the-head style headset then this needs to be Alt.

Also, who is CJ? This does not follow the convention for naming headsets.
```suggestion
  id: ClothingHeadsetAltJustice
```